Sources/Stencil/PlistContext.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  PlistContext.swift
+//  SwiftGenKit
+//
+//  Created by Toshihiro Suzuki on 4/12/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import Foundation
+extension PlistParser {
+  public func stencilContext() -> [String: Any] {
+    return ["plistKeys": keys]
```

Comment:
See comment above: try to support multiple files, and again: we're in a plist parser, don't repeat that information.